inference-engine/src/legacy_api/src/net_pass.cpp

```diff
@@ -953,6 +953,8 @@ static bool unrollLSTMCellBody(CNNLayerPtr cur) {
     auto sum = _eltw(name + ":sum", prc, {N, S}, "sum");
     auto mul = _eltw(name + ":mul", prc, {N, S}, "mul");
 
+    auto pwr = _pwr(name + ":sum_out", prc, {N,S}, 1.0f, 0.0f);
```

Comment:
Adding this here means the layer will be inserted for all plugins - I don't think this is what we want.

Reply:
Right, I'll make it so it happens for GNA only, and only if LSTMCell is the last layer

Reply:
Done in plugin-only